content/en/basics/how-retrieval-works/basic-retrieval/index.md

```diff
@@ -90,29 +54,29 @@ This command uses a `|` to chain two commands together. This will work on Linux
 An example of fetching and extracting a single file, identified by its CID:
 
 ```shell
-lassie fetch -o - bafykbzaceatihez66rzmzuvfx5nqqik73hlphem3dvagmixmay3arvqd66ng6 | car extract - > lidar-data.tar
+lassie fetch -o - bafykbzacecjedcvniq5wylq7cqre7la6diaxwsue5ssy3f3rzftwu3ielspru | car extract - > Juantalks.mp4
```

Comment:
What's the benefit of using a talk from Juan?

Reply:
@smagdali You are probably better suited to answer this as well. Although, Juantalk is very slow to retrieve and is available via only 1 source over single protocol.

Reply:
I don't want this to get left behind the merge window, so I'm gonna revert this back to the LIDAR data. If there's a pressing reason to use a talk, create a new issue for it.